pyuvdata/uvcal.py

```diff
@@ -190,6 +196,41 @@ def __init__(self):
                                            description=desc, form=(2,),
                                            expected_type=float, tols=1e-3)
 
+        # --- cal_style parameters ---
+        desc = ('Style of calibration. Values are sky or redundant.')
```

Comment:
This is already defined above. Should it be defined again here?

Reply:
oops! thank you! I'll remove one of them

Reply:
I took out the upper one.